ui/apps/platform/src/Containers/Vulnerabilities/NodeCves/Overview/NodeCvesOverviewPage.tsx

```diff
@@ -82,8 +87,21 @@ function NodeCvesOverviewPage() {
             entityTab === 'CVE' ? cveDefaultSortOption : nodeDefaultSortOption,
             'replace'
         );
+
+        analyticsTrack({
+            event: NODE_CVE_ENTITY_CONTEXT_VIEWED,
+            properties: {
+                type: entityTab,
+                page: 'Overview',
+            },
+        });
     }
 
+    // Track the current entity tab when the page is initially visited.
+    useEffect(() => {
+        onEntityTabChange(activeEntityTabKey);
+    }, []);
+
```

Comment:
Ideally we would be able to hook into a navigation event to fire this analytics call, but it isn't apparent how to do this with react-router-v5 in a clean way without pushing the concern to a higher level than the "Vulnerabilities" container.

It might be worthwhile to see if this is something that can be handled at the top level "Page" component for the container instead.